services/tables/src/main/java/com/linkedin/openhouse/tables/services/TablesServiceImpl.java

```diff
@@ -106,19 +106,22 @@ public Pair<TableDto, Boolean> putTable(
     // FIXME: save method redundantly issue existence check after findById is called above
     TableDto tableDtoToSave =
         tablesMapper.toTableDto(
-            tableDto.orElse(
-                TableDto.builder()
-                    .tableUri(
-                        TableUri.builder()
-                            .tableId(tableId)
-                            .databaseId(databaseId)
-                            .clusterId(createUpdateTableRequestBody.getClusterId())
-                            .build()
-                            .toString())
-                    .tableUUID(
-                        tableUUIDGenerator.generateUUID(createUpdateTableRequestBody).toString())
-                    .tableCreator(tableCreatorUpdater)
-                    .build()),
+            tableDto.orElseGet(
```

Comment:
The essential change is just `orElse` -> `orElseGet`. Somehow spotless decide to do a full-blown format change.